internal/cli/ex-get-slots.go

```diff
@@ -0,0 +1,58 @@
+package cli
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	"github.com/coreos/airlock/internal/lock"
+	"github.com/spf13/cobra"
+)
+
+var (
+	// cmdGetSlots holds `airlock ex get slots`
+	cmdGetSlots = &cobra.Command{
+		Use:   "slots",
+		Short: "Introspect groups/slots state",
+		RunE:  runGetSlots,
+	}
+)
+
+// runGetSlots performs live introspection of groups/slots.
+func runGetSlots(cmd *cobra.Command, cmdArgs []string) error {
+	if runSettings == nil {
+		return errors.New("nil runSettings")
+	}
+
+	for group, maxSlots := range runSettings.LockGroups {
```

Comment:
Worth doing an early check if `group == ""` to skip fetching the manager / semaphore (as `printHumanShort` will just immediately exit anyways)?